arkouda/dtypes.py

```diff
@@ -63,13 +63,13 @@ def translate_np_dtype(dt: np.dtype) -> Tuple[builtins.str, int]:
     kind = trans[dt.kind]
     return kind, dt.itemsize
 
-def resolve_scalar_dtype(val : object) -> str:
+def resolve_scalar_dtype(val : Union[np.bool,np.float64,int,np.int,np.uint8]) -> str: #type: ignore
```

Comment:
This function should accept all scalar types, not just numpy dtypes. Can we change it back to `object`?

Reply:
@reuster986 Gotcha, absolutely, will do

Reply:
@reuster986 fixed